### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by swatigoel**
This is first commit to move tuvali from Mosip

### CODE REVIEW

File: .github/keys/mosipgpgkey_sec.gpg
Comment (tkuhrt) [inline .github/keys/mosipgpgkey_sec.gpg:1]: This appears to be a secret key. Should this be checked in?

Comment (swatigoel) [reply .github/keys/mosipgpgkey_sec.gpg:1]: @tkuhrt yes this is the key to sign the artifact. But this file is encrypted and is not useful until we know the secret to use this file.
This key was used to sign and publish artifact in mosip maven repository. After moving to owf, we can delete this and have keys specific to owf.

**Comment by swatigoel** [comment]
@tkuhrt I have removed workflow and gpg keys. Can you please review this PR?

**Comment by tkuhrt** [comment]
@swatigoel : You do not need Ry or my approval prior to merging this PR. This PR should be reviewed and approved by the @openwallet-foundation-labs/tuvali-maintainers

**Pull request merged**